modules/nw-cudn-cr.adoc

```diff
@@ -0,0 +1,169 @@
+//module included in the following assembly:
+//
+// * networking/multiple_networks/primary_networks/about-user-defined-networks.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-cudn-cr_{context}"]
+= Creating a ClusterUserDefinedNetwork custom resource
+
+The following procedure creates a `ClusterUserDefinedResource` custom resource definition (CRD). Based upon your use case, create your request using either the `cluster-layer-two-udn.yaml` example for a `Layer2` topology type or the `cluster-layer-three-udn.yaml` example for a `Layer3` topology type.
```

Comment:
would it make sense to add a note / disclaimer indicating that for virt use cases the only valid topology is layer2 ? 

quite sure this is mentioned in the virt section of the docs though.

Reply:
Just to be safe, I added "* {VirtProductName} only supports the `Layer2` topology."